lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -987,7 +1027,7 @@ def _get_anonymous_id(self, course_id, xblock_class):
             location=location,
             static_asset_path=None,
             _runtime=Mock(
-                spec=Runtime,
+                spec=DescriptorSystem,
```

Comment:
Because we have contracts that require `DescriptorSystem`, and because this gives a more accurate representation of the available methods.